packages/language-picker/src/constants.ts

```diff
@@ -0,0 +1,294 @@
+/**
+ * Internal dependencies
+ */
+import { LanguageGroup } from './language-picker';
+import { I18nReact } from '@automattic/react-i18n';
```

Comment:
Minor nit pick: using [`import type`](https://www.typescriptlang.org/docs/handbook/modules.html#importing-types) makes it easier for tooling to optimise the case where a package is only geing imported for a type.

```suggestion
import type { LanguageGroup } from './language-picker';
import type { I18nReact } from '@automattic/react-i18n';
```

Not a big deal, it just stood out to me because I thought we had an eslint warning about this. Eslint appears to be working in this new package though so I must be mistaken.